css/main.css

```diff
@@ -413,8 +419,10 @@ h1{
 	margin-left: 0.5em; /* To put space between scroll bar of main panel and this bar to avoid accidental scrolling. */
 }
 .mainPanel{
+	margin: 0 0 0 0.5em; /* Main panel only has a margin on the left so that iframe scroll bars aren't displaced. */
```

Comment:
I think this is probably better done as 
margin: 0;
margin-right: 0.5em;

you can mix and match short hands with specifics, just do the most generalized first.

Reply:
I see what you mean, I thought the point was to keep everything on one line with the shorthands but my eyes trying to read 4 zeros in a row is making it harder to read in this case lol. I'll fix this.